src/structures/Guild.js

```diff
@@ -321,6 +321,15 @@ class Guild {
   }
 
   /**
+   * The client user as a GuildMember of this guild
+   * @type {GuildMember}
```

Comment:
`{?GuildMember}` and remove all the changes not related to this pr, put them in another one